commit 3bc4d4922ac80f551c6f314dab101c828fc492b4
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 10:25:05 2010 -0700

    Staging: brcm80211: typedefs.h: remove uint8 definition
    
    It's no longer needed anywhere, so remove it.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -58,10 +58,6 @@ typedef unsigned long ulong;
 
 /* define [u]int8/16/32/64, uintptr */
 
-#ifndef TYPEDEF_UINT8
-typedef unsigned char uint8;
-#endif
-
 #ifndef TYPEDEF_UINT16
 typedef unsigned short uint16;
 #endif
@@ -110,7 +106,6 @@ typedef signed int int32;
 #undef TYPEDEF_USHORT
 #undef TYPEDEF_UINT
 #undef TYPEDEF_ULONG
-#undef TYPEDEF_UINT8
 #undef TYPEDEF_UINT16
 #undef TYPEDEF_UINT32
 #undef TYPEDEF_UINTPTR
